lib/make-postcss.js

```diff
@@ -1,16 +1,28 @@
+let path = require("path");
 let postcss = require("postcss");
 let atImport = require("postcss-import");
-let { promisify } = require("faucet-pipeline-core/lib/util");
+let postcssPresetEnv = require("postcss-preset-env");
+let { repr, promisify } = require("faucet-pipeline-core/lib/util");
 let readFile = promisify(require("fs").readFile);
 
 module.exports = function(input, target, assetManager, sourcemaps, browsers) {
-	// if(browsers && browsers.length > 0) {
-	// let filepath = path.relative(assetManager.referenceDir, input);
-	// console.error(`compiling CSS ${repr(filepath)} for ${browsers.join(", ")}`);
-
-	let processor = postcss([
+	let plugins = [
 		atImport()
-	]);
+	];
+
+	if(browsers && browsers.length > 0) {
+		let filepath = path.relative(assetManager.referenceDir, input);
+		console.error(`compiling CSS ${repr(filepath)} for ${browsers.join(", ")}`);
+		plugins.push(postcssPresetEnv({
+			stage: 2, // maybe a little too unstable?
```

Comment:
I'm not familiar with the CSS process, but the JavaScript experience suggests that combining volatile (i.e. not-yet-finalized) features in a single package/declaration/whatever is bound to be troublesome:
https://babeljs.io/blog/2018/07/27/removing-babels-stage-presets
(TLDR: What's stage 2 today might be different next month, both in terms of the set of features and their respective contracts. Thus volatile features need to be versioned independently.)

Reply:
I'm discussing this in the above linked issue. I'm not sure what is the best approach yet. But I'm not sure it makes sense to choose features by their stage 🤔